sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/checks/XmlSchemaCheck.java

```diff
@@ -153,7 +157,10 @@ private static Schema createSchema(String[] schemaList) {
       CACHED_SCHEMAS.put(cacheKey, schema);
       return schema;
     } catch (SAXException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)